NasdaqDataLink.cs

```diff
@@ -51,7 +75,7 @@ static NasdaqDataLink()
         /// <summary>
         /// Default <see cref="NasdaqDataLink"/> constructor uses Close as its value column
         /// </summary>
-        public NasdaqDataLink() : this("Close")
+        public NasdaqDataLink()
```

Comment:
>  : this("Close")

This default value should still be handled I think, maybe calling `SetValueColumnName("Close")` in the body?